src/strings/countWords.c

```diff
@@ -0,0 +1,21 @@
+#include<stdio.h>
+int words(char* str);
+
+int main()
```

Comment:
Remove the main function from this file and DON'T USE printf and scanf unless it is necessary.